NAMESPACE

```diff
@@ -137,7 +137,6 @@ export(validateDimension)
 export(validateUnit)
 import(ospsuite.utils)
 import(rClr)
-import(stringr)

```

Comment:
We are using only a single function. No need to import the entire package. Namespacing `::` is enough.